Makefile.toml

```diff
@@ -25,10 +25,12 @@ BUILDSYS_NAME = "bottlerocket"
 # If you're building a Bottlerocket remix, you'd want to set this to something like
 # "Bottlerocket Remix by ${CORP}" or "${CORP}'s Bottlerocket Remix"
 BUILDSYS_PRETTY_NAME = "Bottlerocket OS"
+# SDK name used for building
+BUILDSYS_SDK_NAME="thar-be-a-beta"
 # SDK version used for building
-BUILDSYS_SDK_VERSION="v0.24.0"
+BUILDSYS_SDK_VERSION="v0.25.0"
 # Site for fetching the SDK
-BUILDSYS_REGISTRY="public.ecr.aws/bottlerocket"
+BUILDSYS_REGISTRY="public.ecr.aws/c3k7p7u2"
```

Comment:
I assume this is just for testing?

Reply:
Yep. Once I'm finished up I'll remove that temp commit and mark this PR ready for review.